### PR TITLE
Improve gui

### PR DESCRIPTION
**Pull request by giacomocaironi**
This PR tries to improve the GUI look and usability:
* Improve SignMessageDialog, GetXpubDialog, SignPsbtDialog and MainWindow widget size and spacing
* Add buttons to import a psbt from a file and to export it after signing

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -1 +0,0 @@
-3.9.7
```
Comment (achow101) [inline .python-version:1]: This file shouldn't be removed, it's for pyenv to use a specific python version.

Comment (giacomocaironi) [reply .python-version:1]: I would like to use a virtual environment with pyenv, with this file commited I can't. 

In general .python-version should be commited only for an application, not for a library: https://github.com/github/gitignore/blob/e5323759e387ba347a9d50f8b0ddd16502eb71d4/Python.gitignore#L85-L88. I don't think that devs using pyenv should be locked to a specific python version, if HWI itself supports many

Comment (giacomocaironi) [reply .python-version:1]: Anyway if it is a problem I can revert the change, it was only a nice touch (in my opinion) not essential for the PR

**Comment by giacomocaironi** [comment]
@achow101 Any updates?

File: .gitignore [hunk 2]
```diff
@@ -25,3 +25,5 @@ env.bak/
 venv.bak/
 
 .vscode
+
+.python-version
```
Comment (achow101) [inline .gitignore]: Please put this change in its own commit.

File: hwilib/_gui.py [hunk 3]
```diff
@@ -125,8 +125,25 @@ def __init__(self, client):
...
+                self.ui.psbt_in_textedit.setPlainText(f.read())
+
+    @Slot()
+    def export_psbt_button_clicked(self):
+        filename, _ = QFileDialog.getSaveFileName(self, 'Save file')
+        if filename:
+            with open(filename, 'w') as f:
+                f.write(self.ui.psbt_out_textedit.toPlainText())
+
+
```
Comment (achow101) [inline hwilib/_gui.py]: Introducing new buttons should be in a separate commit.

File: hwilib/_gui.py [hunk 4]
```diff
@@ -303,12 +320,12 @@ def __init__(self, passphrase='', chain=Chain.MAIN):
...
-        self.ui.signmsg_button.setEnabled(False)
-        self.ui.display_addr_button.setEnabled(False)
-        self.ui.getkeypool_opts_button.setEnabled(False)
-        self.ui.toggle_passphrase_button.setEnabled(False)
+        self.ui.getxpub_button.setEnabled(True)
+        self.ui.signtx_button.setEnabled(True)
+        self.ui.signmsg_button.setEnabled(True)
+        self.ui.display_addr_button.setEnabled(True)
+        self.ui.getkeypool_opts_button.setEnabled(True)
+        self.ui.toggle_passphrase_button.setEnabled(True)
```
Comment (achow101) [inline hwilib/_gui.py:327]: I don't think this is correct. We don't want users to be able use these dialogs if no device is selected yet.

File: hwilib/ui/getxpubdialog.ui [hunk 5]
```diff
@@ -6,109 +6,97 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1205</width>
-    <height>158</height>
+    <width>430</width>
+    <height>140</height>
```
Comment (achow101) [inline hwilib/ui/getxpubdialog.ui:10]: xpubs are pretty long, so shrinking this makes the xpub display kind of unergonomic.

File: hwilib/ui/signmessagedialog.ui [hunk 6]
```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>957</width>
-    <height>350</height>
+    <width>660</width>
```
Comment (achow101) [inline hwilib/ui/signmessagedialog.ui:9]: This has caused the close button to be out of the dialog bounds.

**Comment by giacomocaironi** [comment]
The new commit should have fixed everything

**Comment by achow101** [comment]
ACK fbb34a3bcba2873ef60281a48078b81d07a720e1
